apps/examples/rdp/rdp_main.c

```diff
@@ -82,15 +83,25 @@ uint32_t no_protection_func(uint32_t data)
 static int rdp_demo(int argc, FAR char *argv[])
 {
 	int i = 0;
-	uint32_t rdp_result;
-	uint32_t no_rdp_result;
+	security_handle hnd;
+	uint32_t rdp_result, no_rdp_result;
+
+	int res = security_init(&hnd);
+	if (res != 0) {
+		printf("security_init fail\n");
+		return 0;
+	}
 
 	/*
 	 * Tasks are not created with a secure context.
 	 * Any task that is going to call secure functions must call up_allocate_secure_context()
 	 * to allocate itself a secure context before it calls any secure function
 	 */
-	up_allocate_secure_context(CONFIG_MIN_SECURE_STACKSIZE);
+	res = security_allocate(hnd, CONFIG_MIN_SECURE_STACKSIZE);
+	if (res != 0) {
+		printf("security_allocate fail\n");
+		return 0;
```

Comment:
Is it okay not to call the `security_deinit`?

Reply:
yes, it is needed